stand/libsa/stand.h

```diff
@@ -352,7 +352,7 @@ extern ev_sethook_t	env_noset;		/* refuse set operation */
 extern ev_unsethook_t	env_nounset;		/* refuse unset operation */
 
 /* stdlib.h routines */
-extern int		abs(int a);
+extern int		abs(int a) __pure2;
```

Comment:
Also a separate commit. It may be correct, but it's unrelated to the K&R function definitions -> ANSI prototypes that's the main thrust of this change.